src/main/xerus/monstercat/api/Player.kt

```diff
@@ -217,15 +251,26 @@ object Player: FadingHBox(true, targetHeight = 25) {
 	}
 	
 	/** Set the [tracks] as the internal playlist and start playing from the specified [index] */
-	fun playTracks(tracks: List<Track>, index: Int) {
-		playTrack(tracks[index])
-		onFx {
-			if(index > 0)
-				children.add(children.size - 3, buttonWithId("skipback") { playTracks(tracks, index - 1) })
-			if(index < tracks.lastIndex)
-				children.add(children.size - 3, buttonWithId("skip") { playTracks(tracks, index + 1) })
+	fun playTracks(tracks: List<Track>, index: Int = 0) {
+		Playlist.setTracks(tracks)
+		playFromPlaylist(index)
+	}
+	
+	fun playNext(){
+		val next = Playlist.getNext()
```

Comment:
What do you think about this?
```kotlin
Playlist.getNext()?.let { playTrack(it) } ?: reset()
```

You can also do a similar thing for `playFromPlaylist` and `playPrev`

Reply:
f8eea82b4e90ce2272c0ac2e2caca58216262fd6